### DISCUSSION TITLE
Will this work in Vercel?

### DISCUSSION THREAD

**Discussion by karlo-humanmanaged** [expatfile/next-runtime-env#28] [General]
I'm not sure how to get this to work on Vercel

**Comment by HofmannZ** [comment]
When would you need this on Vercel?

**Reply by karlo-humanmanaged** [reply]
not asap but when I looked at this back in May, I was investigating on how we can make have environment variables to get updated on runtime for both serverless and client code in NextJS (deployed in Vercel)

**Reply by HofmannZ** [reply]
Vercel does not support the [Build once, deploy many](https://www.mikemcgarr.com/blog/build-once-deploy-many.html) principle.

You can just change the variables in the Vercel web interface and have it build a new deployment.
